runtime/src/proposals_configuration/staging.rs

```diff
@@ -372,6 +372,21 @@ pub(crate) fn update_token_pallet_token_governance_parameters(
     }
 }
 
+// TODO (luxor/reward-curve-proposal): adjust parameters
+// Proposal parameters for the 'Set Era Payout Damping Factor' proposal
+pub(crate) fn set_era_payout_damping_factor() -> ProposalParameters<BlockNumber, Balance> {
+    ProposalParameters {
+        voting_period: days!(3),
```

Comment:
`days` macro is not imported. But I think for staging profile we probably want minutes here.